frontend/src/components/layout/Layout.module.scss

```diff
@@ -22,6 +22,7 @@
 
 .header {
   position: sticky;
+  top: 0;
```

Comment:
Praise: This is one of those pesky CSS fixes. Nice one-liner to get the sticky sticking. 😵‍💫